### PR TITLE
adding no type default handler to widinfo

### PR DESCRIPTION
**Pull request by tsurdilo**

### CODE REVIEW

**Comment by MarianMacik** [comment]
Failed tests will be fixed after #1172 is merged.

**Comment by mswiderski** [comment]
Jenkins retest this

**Comment by mswiderski** [comment]
Jenkins retest this

File: jbpm-workitems/jbpm-workitems-core/src/main/java/org/jbpm/process/workitem/core/util/WidInfo.java [hunk 1]
```diff
@@ -100,6 +104,14 @@ private String setParamValue(String origVal,
         }
     }
 
+    private String removeType(String value) {
+        if (value != null && value.startsWith("mvel: ")) {
```
Comment (mswiderski) [inline jbpm-workitems/jbpm-workitems-core/src/main/java/org/jbpm/process/workitem/core/util/WidInfo.java]: is this the only type possible? what about reflection?

**Comment by tsurdilo** [comment]
@mswiderski oops forgot that - amended. thanks!

**Pull request merged**